swaybar/i3bar.c

```diff
@@ -21,6 +21,9 @@ void i3bar_block_unref(struct i3bar_block *block) {
 		free(block->full_text);
 		free(block->short_text);
 		free(block->align);
+		if (block->min_width_str) {
```

Comment:
`free(NULL)` is a no-op so the condition here is actually unnecessary and can be removed